docs/docs-content/audit-logs/monitor-with-datadog.md

```diff
@@ -0,0 +1,167 @@
+---
+sidebar_label: "Monitor Clusters with Datadog"
+title: "Monitor Host Clusters with Datadog"
+description: "Learn how to use Datadog to monitor Palette host clusters."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 50
+hiddenFromNav: false
+tags: ["audit-logs", "datadog", "monitoring"]
+---
+
+You can use [Datadog](https://www.datadoghq.com/), a monitoring and analytics tool, to visualize the audit logs of your
+host clusters.
+
+This guide will walk you through the process of configuring Palette to export audit logs from your host clusters to
```

Comment:
I think it would be useful to make it clear that datadog will receive the data and no longer be in Palette only. Maybe add an info box with that?

Reply:
Your comment made me realize that Palette doesn't have the logs, the cluster has. I rewrote the sentence  :white_check_mark: